web/server/entrypoint.go

```diff
@@ -455,8 +462,11 @@ func (s *robotServer) serveWeb(ctx context.Context, cfg *config.Config) (err err
 	ctx, cancel := context.WithCancel(ctx)
 
 	hungShutdownDeadline := 60 * time.Second
+	// "rdk.stack-traces" is listed as a diagnostic logger in app; users will not see
+	// viam-server stack traces by default on app.viam.com.
+	stackTraceLogger := s.rootLogger.Sublogger("stack-traces")
 	slowWatcher, slowWatcherCancel := utils.SlowGoroutineWatcherAfterContext(
-		ctx, hungShutdownDeadline, "server is taking a while to shutdown", s.rootLogger)
+		ctx, hungShutdownDeadline, "server is taking a while to shutdown", stackTraceLogger)
```

Comment:
I think this is ok to keep at the root logger (this only affects the "server is taking a while to shutdown" log, which should be userfacing

Reply:
nvm, you're right, I misread the code